pyproject.toml

```
@@ -0,0 +1,44 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "svgdigitizer"
+license = {text = "GPL 3.0+"}
```

I believe that this should just be the SPDX identifier:
```suggestion
license = "GPL-3.0-or-later"
```